commands.pm

```diff
@@ -140,6 +130,22 @@ sub test_data {
     return $self->render_not_found;
 }
 
+sub get_asset {
+    my ($self) = @_;
+
+    my $path = join '/', $bmwqemu::vars{'ASSETDIR'}, $self->param('assettype'), $self->param('assetname');
```

Comment:
can we guarantee that assettype and assetname is not .. here? Only relpath is checked below